### PR TITLE
Add and fix documentation 

### PR DESCRIPTION
**Pull request by dschwen**
Refs #7909

### CODE REVIEW

File: modules/phase_field/doc/content/documentation/modules/phase_field/Phase_Field_Equations.md [hunk 1]
```diff
@@ -197,104 +197,11 @@ In the split form of the Cahn-Hilliard solution, the two residual equations are
...
-[`DerivativeParsedMaterial`](FunctionMaterials), where the free energy is entered in the input file
-and all required derivatives are taken automatically. This approach is highly encouraged, as it
-drastically simplifies model development.
-
-
 ## See also
 
 - [Phase Field FAQ](FAQ) - Frequently asked questions for MOOSE phase-field models.
-- [Function Materials](FunctionMaterials) - The key component in the modular free energy phase field modeling approach. This page lists the available function materials and explains how to define a free energy function and combine multiple free energy contributions (including elastic energy) into a total free energy.
+- [Actions](phase_field/Actions.md) - Simplify the setup of of teh phase field equations using MOOSE actions
```
Comment (laagesen) [inline modules/phase_field/doc/content/documentation/modules/phase_field/Phase_Field_Equations.md]: typo teh->the

File: modules/phase_field/doc/content/documentation/systems/Kernels/CoupledMaterialDerivative.md [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
 
-!alert construction title=Undocumented Class
-The CoupledMaterialDerivative has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
-
 !syntax description /Kernels/CoupledMaterialDerivative
 
+This kernel addss the first derivative $\frac{\partial F}{\partial v}$ of a given
```
Comment (laagesen) [inline modules/phase_field/doc/content/documentation/systems/Kernels/CoupledMaterialDerivative.md]: typo addss->adds

File: modules/phase_field/doc/content/documentation/systems/Kernels/SimpleCoupledACInterface.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
-
 !syntax description /Kernels/SimpleCoupledACInterface
 
+!include simple_kernels_include.md
+
+This kernel implements the weak form
+\begin{equation}
+(\kappa L \nabla \eta, \nabla\psi)
+\end{equation}
+for the gradient contribution in a couple Allen-Cahn equation.
```
Comment (laagesen) [inline modules/phase_field/doc/content/documentation/systems/Kernels/SimpleCoupledACInterface.md]: couple->coupled

**Review by laagesen — CHANGES_REQUESTED** [review]

**Comment by dschwen** [comment]
@laagesen seems to be pretty swamped. Would you mind taking a look at this @amjokisaari ? I'd like to get this in to avoid conflicts with other doc work.

**Comment by dschwen** [comment]
Oh, darn, should have reloaded the page. I didn't see you commented already @laagesen !

**Review by laagesen — APPROVED** [review]

**Review by aeslaughter — APPROVED** [review]

**Pull request merged**